generators/server/templates/src/main/resources/config/cql/_create-keyspace-prod.cql

```diff
@@ -1,3 +1,13 @@
 CREATE KEYSPACE IF NOT EXISTS <%= baseName %>
     WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 }
     AND DURABLE_WRITES = true;
+
+CREATE TABLE IF NOT EXISTS <%= baseName %>.schema_version (
```

Comment:
My Unit test failed for microservice cassandra.
Maybe because there is no cql file, so no table, so no keyspace created.
What about moving this CREATE TABLE into 0000_create_table like before and remove <%= baseName %>. of course?